src/libraries/System.Utf8String.Experimental/tests/System/Utf8SpanTests.Searching.TestData.cs

```diff
@@ -499,6 +499,7 @@ private static IEnumerable<TryFindTestData> TryFindData_All()
                     ExpectedFirstMatch = null,
                     ExpectedLastMatch = null,
                 },
+#if !NETFRAMEWORK
```

Comment:
@GrabYourPitchforks - these are the 3 cases for `TryFind` that don't work outside of CoreLib. I noticed on https://github.com/dotnet/runtime/issues/29442#issuecomment-546521556 you mentioned potentially throwing outside of `netcoreapp5.0` for culture-sensitive comparisons. I'm not sure how to make it work (see my [TODO here](https://github.com/dotnet/runtime/pull/33357/files#diff-b37c0807428c979414a7a88b49f382f0R287)).